src/spec-node/devContainersSpecCLI.ts

```diff
@@ -48,6 +49,7 @@ const defaultDefaultUserEnvProbe: UserEnvProbe = 'loginInteractiveShell';
 	y.command('run-user-commands', 'Run user commands', runUserCommandsOptions, runUserCommandsHandler);
 	y.command('read-configuration', 'Read configuration', readConfigurationOptions, readConfigurationHandler);
 	y.command(restArgs ? ['exec', '*'] : ['exec <cmd> [args..]'], 'Execute a command on a running dev container', execOptions, execHandler);
+	y.command('merge [args..]', 'Merge a parent and a child dev container config', mergeOptions, mergeHandler);
```

Comment:
I think is this implicitly part of `read-configuration` and it should full-resolve any `extends` references in the configuration.